src/generate-signing-key.ts

```diff
@@ -0,0 +1,11 @@
+import { webcrypto } from 'node:crypto';
```

Comment:
To be honest, this could be a JavaScript file and save us a ts-node. We're not really even using any types here other than the Node API.